client/src/components/TaskStatusChart/TaskStatusChart.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import styles from "./TaskStatusChart.module.css";
+import { PieChart, Pie, Cell, Legend } from "recharts";
+import { PropTypes } from "prop-types";
+
+const RADIAN = Math.PI / 180;
+const renderCustomizedLabel = ({
+  cx,
+  cy,
+  midAngle,
+  innerRadius,
+  outerRadius,
+  percent,
+  index,
+}) => {
+  const radius = innerRadius + (outerRadius - innerRadius) * 0.5;
+  const x = cx + radius * Math.cos(-midAngle * RADIAN);
+  const y = cy + radius * Math.sin(-midAngle * RADIAN);
+
+  return (
+    <text
+      x={x}
+      y={y}
+      fill="white"
+      textAnchor={x > cx ? "start" : "end"}
+      dominantBaseline="central"
+    >
+      {`${(percent * 100).toFixed(0)}%`}
+    </text>
+  );
+};
+
+const COLORS = ["#629632", "#F6BE00", "#FF0000"];
+
+export const TaskStatusChart = ({ tempData }) => {
```

Comment:
I would also include a header of some sort so we know what each chart is supposed to represent

![image](https://user-images.githubusercontent.com/24628243/164571128-e355a1b6-603c-438b-be7e-58b80e27e9b7.png)

Reply:
Resolved: I made this change.